packages/terra-section-header/src/SectionHeader.scss

```diff
@@ -0,0 +1,92 @@
+@import '~terra-mixins';
+
+:local {
+  .section-header {
+    @include terra-padding-end(var(--terra-section-header-padding-end, 0.7142857143rem));
+    @include terra-padding-start(var(--terra-section-header-padding-start, 0.7142857143rem));
+   
+    background: var(--terra-section-header-background, #e8e9ea);
+    border-bottom: var(--terra-section-header-border-bottom, 1px solid #c8cacb);
+    border-left: var(--terra-section-header-border-left);
+    border-right: var(--terra-section-header-border-right);
+    border-top: var(--terra-section-header-border-top);
+    box-shadow: var(--terra-section-header-box-shadow);
+    color: var(--terra-section-header-color, #1c1f21);
+    line-height: var(--terra-section-header-line-height, 1.142857143rem);
+    padding-bottom: var(--terra-section-header-padding-bottom, 0.5rem);
+    padding-top: var(--terra-section-header-padding-top, 0.5rem);
+    width: 100%;
+
+    &.is-interactable {
+      @include terra-padding-start(var(--terra-section-header-accordion-padding-start, 0.5rem));
+      cursor: pointer;
+
+      &:hover {
+        background: var(--terra-section-header-hover-background, #d7d8d9);
+        border-bottom: var(--terra-section-header-hover-border-bottom);
+        border-left: var(--terra-section-header-hover-border-left);
+        border-right: var(--terra-section-header-hover-border-right);
+        border-top: var(--terra-section-header-hover-border-top);
+        box-shadow: var(--terra-section-header-hover-box-shadow);
+        color: var(--terra-section-header-hover-color);
+      }
+
+      &:focus {
+        background: var(--terra-section-header-focus-background, #e8e9ea);
+        border-bottom: var(--terra-section-header-focus-border-bottom);
+        border-left: var(--terra-section-header-focus-border-left);
+        border-right: var(--terra-section-header-focus-border-right);
+        border-top: var(--terra-section-header-focus-border-top);
+        box-shadow: var(--terra-section-header-focus-box-shadow, 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35));
```

Comment:
Is there a cutoff/standard when we are ok with using px rather than rems?

Reply:
It depends on the situation, not all styles are going to be relative to the font-size. 
So far, style rules we generally do not make specifically **rem **based are: Border-widths, Box-shadows, widths/min-widths, SVG's height & width (doesn't accept rem).